Migrate widgets constants to use getter/setter style (#4972)
DFHack/dfhack: docs/dev/Lua API.rst

its minimum size. Both jumps respect the resizable edges defined by the
``resize_anchors`` attribute.

> The time duration that a double click can span can be controlled by widgets API:

Reviewer: Could you update this to indicate that the `control-panel` or `gui/control-panel` interfaces should be used to set these properties? That goes for the others as well. You can remove the `widget.*` function references. These docs were written before the control panel was a thing.

The `gui/control-panel` interface looks like this:
![image](https://github.com/user-attachments/assets/2ce3195e-b397-450a-b050-e28cf7593368)

and the `control-panel` interface looks like this:
![image](https://github.com/user-attachments/assets/22b41f5b-32ec-457e-a2f3-754171eaa100)

the functions should no longer be called by anything other than the control panel interfaces

its minimum size. Both jumps respect the resizable edges defined by the
``resize_anchors`` attribute.

> The time duration that a double click can span default value is ``500``.
> It can be controlled via the `control-panel` or `gui/control-panel` interfaces 
> (``Mouse double click speed`` option).

Reviewer: ```suggestion
The time duration that a double click can span can be controlled
via the `control-panel` or `gui/control-panel` interfaces (``Mouse
double click speed`` option). It defaults to 500 ms.
```

You can click and drag the scrollbar to scroll to a specific spot, or you can
click and hold on the end arrows or in the unfilled portion of the scrollbar to
scroll multiple times, just like in a normal browser scrollbar. The speed of
scroll events when the mouse button is held down can be controlled
via the `control-panel` or `gui/control-panel` interfaces:

> 1. The delay before the second scroll event use
> ``Mouse initial scroll repeat delay`` option (default is 300)

Reviewer: ```suggestion
1. The delay before the second scroll event is the ``Mouse initial scroll repeat
   delay`` setting (default is 300 ms)
```

> 2. The delay between further scroll events use
> ``Mouse scroll repeat delay`` option (default is 20)

Reviewer: ```suggestion
2. The delay between further scroll events is the ``Mouse scroll repeat delay`` option
   (default is 20 ms)
```
